tests/unit/view_execution_tests.py

```diff
@@ -480,8 +480,48 @@ def test_stale_update_after(self):
         try:
             self.view001(stale='update_after')
         except Exception as err:
-            self.assertFail(str(err), 'An unexpected error was encountered.')
+            self.fail('An unexpected error was encountered:' +str(err))
+            
+    def test_stable_true(self):
```

Comment:
Do these tests need to be tagged for Couch2x/Cloudant because of the new parameters? I guess Couch 1x probably ignores them?

Reply:
They seem to be ignored, at least by couch 1.7

Reply:
Since Travis is passing, I'm assuming CouchDB 1.7.1 is ignoring this new param test.